### PR TITLE
chore: remove reference to jina ai

### PR DESCRIPTION
**Pull request by samsja**
# Context

This pr remove the links the Jina AI for the LF AI donation

### CODE REVIEW

**Review by NicholasDunham — APPROVED** [review]

**Review by JoanFM — CHANGES_REQUESTED** [review]
Let's wait until LFAI acceptance.

File: docs/get-started/what-is.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ In DocArray, the basic element you work with is a Document, not `ndarray`. The s
 
 ## To Jina Users
 
-DocArray focuses on local and monolith developer experience. Jina scales DocArray to the Cloud. More details on their relations can be {ref}`found here<jina-support>`.
+DocArray focuses on local and monolith developer experience. Jina scales DocArray to the Cloud.
```
[the remark this answers is not in the document]
Comment (samsja) [reply docs/get-started/what-is.md:122]: yeah my bad

File: docs/index.md [hunk 2]
```diff
@@ -47,7 +47,6 @@ advanced/document-store/index
 :caption: Integrations
 :hidden:
 
-fundamentals/jina-support/index
 fundamentals/notebook-support/index
```
Comment (hanxiao) [inline docs/index.md:50]: keep this

Comment (hanxiao) [inline docs/index.md:50]:
```suggestion
fundamentals/jina-support/index
fundamentals/notebook-support/index
```

**Review by hanxiao — CHANGES_REQUESTED** [review]
~~Keep "Jina support" section~~
actually, never mind

**Review by JoanFM — APPROVED** [review]

**Pull request merged**